src/master-playlist-controller.js

```diff
@@ -256,7 +262,17 @@ export class MasterPlaylistController extends videojs.EventTarget {
     // combined audio/video or just video when alternate audio track is selected
     this.mainSegmentLoader_ = new SegmentLoader(segmentLoaderOptions);
     // alternate audio track
+    segmentLoaderOptions.loaderType = 'audio';
     this.audioSegmentLoader_ = new SegmentLoader(segmentLoaderOptions);
+
+    this.decrypter_.onmessage = (event) => {
+      if (event.data.source === 'main') {
+        this.mainSegmentLoader_.handleDecrypted_(event.data);
+      } else {
```

Comment:
Just preference, but it might be good to keep this as `else if (event.data.source === 'audio')`